src/main/scala/pipelines/images/cifar/RandomCifar.scala

```diff
@@ -41,18 +47,10 @@ object RandomCifar extends Serializable with Logging {
         ImageVectorizer andThen
         new Cacher[DenseVector[Double]] andThen
         (new StandardScaler, trainImages) andThen
-        new Cacher[DenseVector[Double]]
-
-    val labelExtractor = LabelExtractor andThen
-      ClassLabelIndicatorsFromIntLabels(numClasses) andThen
-      new Cacher[DenseVector[Double]]
-
-    val trainFeatures = featurizer(trainImages)
-    val trainLabels = labelExtractor(trainData)
-
-    val model = LinearMapEstimator(conf.lambda).fit(trainFeatures, trainLabels)
+        new Cacher[DenseVector[Double]] andThen
+        (LinearMapEstimator(conf.lambda), trainImages, trainLabels)
 
-    val predictionPipeline = featurizer andThen model andThen MaxClassifier
+    val predictionPipeline = featurizer andThen MaxClassifier
```

Comment:
can we add the `MaxClassfier` to the end of what we're calling `featurizer` and change that name to `predictionPipeline`?